### DISCUSSION TITLE
Using IP instead of FQDN to access the API

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#912] [Q&A] [answered]
Hi

How do I login using ip instead FQDN ?

**Comment by jshcodes** [comment, marked as answer]
Hi @NSH531 -

You can provide an IP address for the `base_url` keyword instead of the FQDN.  

A couple of concerns with this scenario:

- You will need to disable SSL verification with the API, as certificates are not IP aware.
- IPs can be dynamic, so if the IP for the FQDN changes your connections to the API will fail.

With those notes in mind, this can be accomplished using the following syntax:

```python
from falconpy import Hosts

hosts = Hosts(client_id=CLIENT_ID,
              client_secret=CLIENT_SECRET,
              base_url=IP_ADDRESS,
              ssl_verify=False
              )
```

**Reply by NSH531** [reply]
How to do this with swagger?

**Reply by jshcodes** [reply]
This would not be supported via swagger.

**Reply by NSH531** [reply]
Do you can add supporting this using swagger in next release?

Nate
